src/Event/HttpRequestBuilt.php

```diff
@@ -17,7 +17,7 @@ public function __construct(
         $this->httpRequest = $httpRequest;
     }
 
-    public function &get(): RequestInterface
+    public function get(): RequestInterface
```

Comment:
No need to expressly return these by reference, as PHP inherently passes objects by reference.

Reply:
would this be a breaking change?

Reply:
Nope! The result is the same and compatible with any previous iterations (although this class is new to 8.0, anyway.) It was just a redundant language instruction; & tells PHP to pass something by reference rather than value, such as an array value; but objects, such as this case, are always passed by reference, so it was just unnecessary.